Test/test_cmor_license_template.py

```diff
@@ -0,0 +1,161 @@
+import json
+import cmor
+import unittest
+import os
+import numpy
+
+from netCDF4 import Dataset
+
+LICENSE_INFO = {
+    "license_template":
+        ("CMIP7 data produced by <institution_id> is licensed under a <license_type> "
+        "License (<license_url>). Consult https://pcmdi.llnl.gov/CMIP6/TermsOfUse "
```

Comment:
We also need to define a `TermsOfUse` landing page - @wolfiex, can we do this with GH pages and CMIP7-CVs - https://wcrp-cmip.github.io/CMIP7-CVs/TermsOfUse?

Reply:
@durack1 Did we want to update the URL to https://pcmdi.llnl.gov/CMIP7/TermsOfUse with the page being set up on a later date?

Reply:
@mauzey1 good catch. I wonder whether we direct to https://wcrp-cmip.github.io/cmip7-cvs (not live), or whether we could get a wcrp-cmip.org landing page stood up @wolfiex?

Reply:
The `TermsOfUse`URL is currently set to https://pcmdi.llnl.gov/CMIP7/TermsOfUse. I'll merge this PR and we can revisit the "Terms of Use" page issue later.